netlify/settings.yaml

```
@@ -136,15 +156,15 @@ header:
     - href: /demo/
       isButton: true
       title: Request Demo
```

Google Optimize results suggest we update Request Demo -> Book a Demo

```suggestion
      title: Book a Demo
```